plugins/acss-kit/assets/foundation/foundation.css

```
@@ -0,0 +1,1586 @@
+  }
+
+  :root {
+    --weight: 600;
+    --fs: clamp(0.96rem, 1vw + 0.5rem, 1.6rem);
+    --p: 1rem;
+    --px: var(--p);
+    --py: var(--p);
+    --m: 0.75rem;
+    --mx: var(--m);
+    --my: var(--m);
+    --weight: 600;
```

Stylelint (17.10.0)

[error] Duplicate custom property "--weight" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)